### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by dlehman83**
Most work in Graylog and SNMP
Fixed many code tags, spelling and wording.
Added SNMP PowerShell example.

Is there a standard for spelling? I saw several words with both UK and US spellings.  I tried not to change these.  

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Support/SNMP-Configuration-Examples.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ path: blob/master/doc/
 
 # SNMP configuration examples
 
-Table of Content:
+[Table of Content:
```
Comment (Jellyfrog) [inline doc/Support/SNMP-Configuration-Examples.md:6]: This seems wrong

File: doc/Support/SNMP-Configuration-Examples.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ Table of Content:
   - [Linux (snmpd v2)](#linux-snmpd)
   - [Linux (snmpd v3)](#linux-snmpd-v3)
   - [Windows Server 2008 R2](#windows-server-2008-r2)
-  - [Windows Server 2012 R2 and 2016](#windows-server-2012-r2-and-2016)
-  - [Mac OSX](#Mac-OSX)
+  - [Windows Server 2012 R2 and newer](#windows-server-2012-r2-and-newer)
+  - [Mac OSX](#Mac-OSX)]
```
Comment (Jellyfrog) [inline doc/Support/SNMP-Configuration-Examples.md:34]: And this

**Comment by dlehman83** [comment]
Thanks for catching that.  I tried switching to [TOC]  instead of writing it all out, but didn’t want to make that large of a change.  It added several sub headings that were not in the manually written TOC.  
Then I missed removing the brackets when reverting.  I’ve pushed the fix to my branch.

**Review by Jellyfrog — APPROVED** [review]

**Review by Jellyfrog — APPROVED** [review]

**Comment by Jellyfrog** [comment]
Thanks

**Pull request merged**